app/models/post.rb

```diff
@@ -0,0 +1,7 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+
+  validates :title, presence: true, length: { maximum: 255 }
+  validates :body, :user_id, presence: true
```

Comment:
- [ ] Kindly validate the scenario where the title is empty.
- [ ] Kindly validate the scenario where the body is empty.